cpucounters.cpp

```diff
@@ -6335,9 +6337,11 @@ uint64 ServerPCICFGUncore::computeQPISpeed(const uint32 core_nr, const int cpumo
                } while (endTSC - startTSC < 200000ULL); // spin for 200 ms
 
                uint64 endClocks = getQPIClocks((uint32)i);
+#if defined(__linux__) || defined(_MSC_VER)
```

Comment:
could you please find a way to avoid this ifdef here and hide it inside the cleanMemTest (similar to initMemTest)? This would make the init/cleanup code symmetric and cleaner.

Reply:
thanks for the updated change. Could you please drop this ifdef?